akka-analytics-kafka/build.sbt

```diff
@@ -5,12 +5,13 @@ fork in Test := true
 parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
-  "com.github.krasserm" %% "akka-persistence-kafka" % "0.4",
-  "com.github.krasserm" %% "akka-persistence-kafka" % "0.4" % "test" classifier "tests",
-  "org.apache.curator"   % "curator-test"           % "2.7.1" % "test",
-  "org.apache.spark"    %% "spark-streaming"        % "1.4.0",
-  "org.apache.spark"    %% "spark-streaming-kafka"  % "1.4.0",
-  "org.apache.kafka"    %% "kafka"                  % "0.8.2.1" excludeAll(
+  "com.typesafe.akka"   %% "akka-persistence-experimental" % "2.3.4",
```

Comment:
There is a dependency Akka Analytics Kafka -> Akka Persistence Kafka -> Akka Testkit -> Akka Persistence (old version). Before updating Akka Analytics Kafka Akka Testkit and Akka Persistence Kafka would need updating too.